packages/scandipwa/src/component/ProductBundleOptions/ProductBundleOptions.style.scss

```diff
@@ -30,10 +30,7 @@
             }
 
             &_type_select + .Field-ErrorMessages {
-                position: absolute;
-                inset-inline-start: 0;
-                inset-block-end: 10px;
-                transform: translateY(100%);
+                margin-block-start: 6px;
             }
```

Comment:
I believe the `position: absolute` was there in order to display the error right under the field without moving other contents down.
Did you check it?

Reply:
Yes, I checked. There was no reason for that element to be postion absolute. I went with the natural div flow and gave a margin, that's more natural way rather it positioned absoutely. I check with the Alex

Reply:
I see, returned back position absolute and considered spacing